src/command_modules/azure-cli-registry/README.md

```diff
@@ -0,0 +1,194 @@
+# Microsoft Azure CLI 'registry' Command Module
+
+## Commands to manage Azure container registries
+```
+Group
+    az registry: Commands to manage Azure container registries.
+
+Commands:
+    catalog: The catalog of containers in the specified registry.
```

Comment:
is container the right word here ?  I thought catalog lists the 'repositories' or 'images', no?